src/commands/build.ts

```diff
@@ -60,7 +60,7 @@ export class BuildCommand implements Command {
       return options.env.build(buildOptions, config);
     }
 
-    logger.debug('building via build()...');
+    logger.debug('building via standard build()...');
```

Comment:
yea I've had trouble with this naming, would love some input. It's the built-in `build` function, as apposed to the `options.env.build` function you can pass via options.